dockerfiles/Dockerfile-toolchain

```diff
@@ -121,13 +113,16 @@ RUN eval $(opam env)
 RUN opam switch import opam.export ; rm opam.export
 
 # Pin packages for vanilla compiler
-RUN cd /ocaml-sodium && opam pin -y add .
-RUN cd /rpc_parallel && opam pin -y add .
-RUN cd /ocaml-extlib && opam pin -y add .
-RUN cd /digestif && opam pin -y add .
-RUN cd /async_kernel && opam pin -y add .
-RUN cd /coda_base58 && opam pin -y add .
-RUN cd /graphql_ppx && opam pin -y add .
+RUN cd $EXTERNAL_PKG_DIR/ocaml-sodium && opam pin -y add .
```

Comment:
Can we run these all as one command? I  think that makes fewer layers with intermediate useless opam libraries from eg the async_kernel pin.

Reply:
I can make a loop. 

The tradeoff is that if one of the later commands fails, you have to execute the whole loop again, instead of having an earlier layer to start from. They should all succeed, of course.

Reply:
Added the loops.